### DISCUSSION TITLE
Unable to apply for GitHub

### DISCUSSION THREAD

**Discussion by minhaalex** [community/community#47736] [GitHub Education]
### Select Topic Area

Question

### Body

As an undergraduate student, I rely on Canva Pro to create featured images, logos, cover photos, and other visuals for [websites](https://dreamifiel.com/food-dreams/). Since I'm on a tight budget, I applied for the free Canva Pro student package. Unfortunately, I encountered an error message stating that "I need to use a school-issued email address to apply". Despite adding my school email and selecting it, I'm still unable to submit my application. It's frustrating, as I really need access to Canva Pro for my school projects.

Thanks looking for a better reply.

**Comment by jge162** [comment]
Hello @minhaalex Sorry to hear you are running into issues. You can possibly resolve your issue by reaching out to [GitHub Education support](https://support.github.com/contact/education). This something we can only discuss here, but support can assist you.

**Reply by minhaalex** [reply]
> Hello @minhaalex Sorry to hear you are running into issues. You can possibly resolve your issue by reaching out to [GitHub Education support](https://support.github.com/contact/education). This something we can only discuss here, but support can assist you.

Thanks for your suggestion

**Reply by jge162** [reply]
Most people have been able to get it fixed through support FYI @minhaalex

**Reply by minhaalex** [reply]
Sure, I contacted there for my fix
